docs/Ruleset.md

```diff
@@ -70,6 +70,8 @@ V096 | PATCH | Class private method parameter default removed
 V097 | MAJOR | Class public method parameter default value changed
 V098 | MAJOR | Class protected method parameter default value changed
 V099 | PATCH | Class private method parameter default value changed
+V150 | PATCH | Class public/protected/private method renamed (case only)
```

Comment:
This should be split into 3 rules, one for each visibility level (as it is done for all other rules even if they are all the same value).